packages/dashboard/src/components/iot-dashboard/iot-dashboard-messages/messages.ts

```diff
@@ -0,0 +1,42 @@
+export type ContextMenuMessages = {
```

Comment:
get rid of the `iot-dashboard-messages` folder - we aren't naming a bunch of folders `iot-NAME` because we want the iot prefix everywhere - it's the pattern for declaring a component, if there's many files and subcomponents, to put it all in a folder that matches the name of the component. In this case I think `messages` should just be moved out of the `components` folder hierarchy completely. The entire package is regarding dashboard, i think it could just go in `src/messages.ts`

Reply:
👍